build/repo.targets

```diff
@@ -154,22 +122,26 @@
       <Solution>
         <AdditionalProperties>RestoreGraphOutputPath=$(_RestoreGraphSpecsDirectory)%(Solution.Repository)\%(Solution.FileName)%(Solution.Extension).json</AdditionalProperties>
       </Solution>
+
+      <GraphSpecInputs Include="
```

Comment:
> If there are any changes in the repo-to-repo graph, a bad cache will fail to build, right?

Unfortunately not. You would get an incorrect build if we have a cache that's gotten in a state where it no longer reflects the current build order. But the issue I pointed out was with us over-doing the task. Plus we could fine tune this if we see local incremental builds (which is the only thing that should be affected by this) get in a bad state.